app/main.py

```diff
@@ -1,6 +1,15 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result_cache = {}
+
+    def inner(*args, **kwargs) -> Any:
+        if args in result_cache:
```

Comment:
what if two different functions have the same `args` but different `kwargs`?
you must use `args` + `kwargs` as the keys of `result_cache `